JUSTFILE

```diff
@@ -93,6 +93,9 @@ load-pytest-kind:
 deploy-tests-helm-ci target:
   helm upgrade --wait --timeout={{TIMEOUT}} --install --values tests/deploy/values/ci.yml --set testData.marker={{target}} --namespace kube-system {{TESTS_RELEASE}}  tests/deploy/k8s-cloudbeat-tests/
 
+deploy-tests-helm-ci-agent target:
```

Comment:
set image versions via --set $(make get-version)-SNAPSHOT to make it branch agnostic

Reply:
Updated JUSTFILE commands - added VERSION as variable and updated all helm commands with set imageTag version.